### ISSUE TITLE
[Feature]: switch between voice and video calls

### ISSUE THREAD

**Issue by ElsAr4e** [simplex-chat/simplex-chat#2860]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### Feature

Please allow that you can switch from an audio-only call to a video call (and the other direction too).

Except if the other user does not allow video calls (but allows audio calls). Which is not possible until now - but you can allow audio-only calls or both audio and video calls(https://github.com/simplex-chat/simplex-chat/issues/2164#issuecomment-1566181116).

If both -audio and video calls -are allowed the user should be able to switch between switch between voice and video calls (including screen sharing (https://github.com/simplex-chat/simplex-chat/issues/2165)) like it is possible using Threema: https://threema.ch/en/faq/calls_disable

**Comment by ElsAr4e** [comment]
And if you the other user suddenly allows video calls (previously just audio-only calls) should this change anything during the same call?

**Comment by jittygitty** [comment]
Yes, it is very annoying with apps (like Element) where I have to "hang up" and start a call again to switch from just audio to a video call, would be nice to "add video" etc. 

I don't know how a phone's regular "video call" automatically drops the video and switches to an audio only call during bad network connection etc. But I don't think even they allow starting as audio call and switching to video mode, or some do?

Now there could be TWO modes, let's way I want to 'add' a video stream on an audio-call like to stream a video file from my device to the other user, or sending a live video from camera but keeping audio separate, this way might be simple since the audio can be kept on original audio stream without caring about "audio sync". 

Now when you want the other person to see your face talking, you probably want the audio muxed into the video stream and synced which is probably harder than having two separate un-synced streams, one video and one audio, as in the example above.
